src/components/LineChart/index.tsx

```diff
@@ -0,0 +1,177 @@
+import { useTheme } from '@emotion/react'
+import { DatumValue, ValueFormat } from '@nivo/core'
+import { LineSvgProps, Point, ResponsiveLine, Serie } from '@nivo/line'
+import PropTypes from 'prop-types'
+import React, { FunctionComponent, Validator, useState } from 'react'
+import CustomLegend, { Transformer } from './CustomLegend'
+import LineChartTooltip from './Tooltip'
+import { getLegendColor, getMaxChartValue, getMinChartValue } from './helpers'
+
+type Formater = ValueFormat<DatumValue>
+
+type Shape<K extends string, T> = {
+  [P in K]?: T
+}
+
+type LineChartProps = {
+  height?: string | number
+  margin?: Shape<'bottom' | 'left' | 'right' | 'top', number>
+  xScale: LineSvgProps['xScale']
+  yScale: LineSvgProps['yScale']
+  data: LineSvgProps['data']
+  withLegend?: boolean
+  axisFormatters?: Shape<'bottom' | 'left' | 'right' | 'top', Transformer>
+  pointFormatters?: Shape<'x' | 'y', Formater>
+  tickValues?: Shape<'bottom' | 'left' | 'right' | 'top', number | string>
+}
+
+const LineChart: FunctionComponent<LineChartProps> = ({
+  height,
+  margin,
+  xScale,
+  yScale,
+  data,
+  withLegend,
+  axisFormatters,
+  pointFormatters,
+  tickValues,
+}) => {
+  const theme = useTheme()
+  const hasData = !data.some(d => !d.data.length)
+  const dataset = {
+    datasets: data.map((d, i) => ({
+      data: hasData ? d.data : [],
+      id: d.id,
+      label: d.label as string,
+      serieColor: getLegendColor(i, theme),
+    })),
+  }
+
+  const chartTheme = {
+    axis: {
+      ticks: {
+        line: {
+          stroke: theme.colors.gray300,
+          strokeWidth: 1,
+        },
+      },
+    },
+    fontFamily: 'Asap',
```

Comment:
We gonna change our font Family so is it posisble to get font family front theme instead ?

Reply:
`theme.fonts.sansSerif`